Dockerfile

```diff
@@ -2,11 +2,11 @@
 FROM golang:1.12.5 as builder
 
 WORKDIR /workspace
-# Copy the Go Modules manifests
-COPY go.mod go.mod
-COPY go.sum go.sum
+
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
+COPY go.mod go.mod
```

Comment:
Maybe I'm missing something here, but it looks to me like only the comment was changed here, wasn't it?

Reply:
This change is not required to fix the build, I just restructured for aesthetic reasons.